src/coreclr/jit/fgopt.cpp

```diff
@@ -543,10 +546,14 @@ bool Compiler::fgRemoveUnreachableBlocks()
 // Also, compute the list of return blocks `fgReturnBlocks` and set of enter blocks `fgEnterBlks`.
 // Delete unreachable blocks.
 //
+//  Arguments:
+//       computeDoms - Whether to compute doms or not.
```

Comment:
It would be useful to understand (and add to this comment block) when renumbering is required, and why. And when it is not. Same for creating doms.